ConsentViewController/Classes/SourcePointClient/SPNativeUI.swift

```diff
@@ -198,7 +198,11 @@ class SPNativeLongButton: SPNativeUI {
             onText = try container.decode(String.self, forKey: .onText)
             offText = try container.decode(String.self, forKey: .offText)
             customText = try container.decode(String.self, forKey: .customText)
-            text = try container.decode(String.self, forKey: .text)
+            do {
```

Comment:
Why is this necessary?

Reply:
It's the reason of crash in case if "text" key is not present in json

Reply:
@andresilveirah okay this change is not actual since we do have container.decodeIfPresent in develop branch

Reply:
@andresilveirah this change is reversed